components/page/body/tabs.vue

```diff
@@ -48,13 +46,32 @@
     const did           = computed(()=> pageStore?.page?.drupalInternalNid);
 
 
+    const baseUrl   = computed(()=>siteStore.localizedHost+getUrlComponent());
+
+    const showSelf = computed(() => pageStore?.isSystemPage? meStore?.showEditSystemPages : meStore?.showEdit)
+
+    const isContributor = computed(() => meStore?.roles?.includes('contributor') && meStore?.roles?.length == 1);
+
+    const isContributorCanEdit = computed(() => isContributor.value && pageStore?.page?.uid?.meta?.drupal_internal__target_id === meStore?.diuid && !pageStore?.page?.status);
+
+    const cloneUrl = computed(()=>siteStore.localizedHost+`/clone/${pageStore?.page?.drupalInternalNid}/quick_clone`+returnUrl.value);
```

Comment:
what returnUrl value contains?
`?returnUrl=somethings..?`. the name is not reflecting the value thosu shoudl be only an url